### PR TITLE
[11.0][MIG] stock_picking_filter_lot

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Migration to v11:

* Adapted to new stock.move.line model.

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

File: stock_picking_filter_lot/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+To use this module:
+
+#. Go to an open picking with products that are tracked by lots:
```
Comment (SimoRubi) [inline stock_picking_filter_lot/readme/USAGE.rst]: Remove semicolon at the end of line

File: stock_picking_filter_lot/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+#. When you enter detailed operations you will notice that the lots available
+   to select are filtered by the origin of the operation.
```
Comment (SimoRubi) [inline stock_picking_filter_lot/readme/USAGE.rst:5]: Why filtered by the origin of the operation?
The module in v10 has the field `filter_location_id` that lets the user specify the location to use in the filter

**Review by SimoRubi — CHANGES_REQUESTED** [review]
Thanks for your contribution!
Just code review waiting for runbot's build

File: stock_picking_filter_lot/readme/USAGE.rst (hunk 2 above)
Comment (LoisRForgeFlow) [reply stock_picking_filter_lot/readme/USAGE.rst:5]: Yes, but why is that needed, if you are moving from a given location you just want to see available lots at that location, why an extra field to filter?

BTW, I was thinking to add pair of improvements:
* Show the the qty available to transfer with a given lot and location filled.
* Add an option on the picking types to remove this filtering, allowing you to force the use of a given lot.

What do you think?

Comment (SimoRubi) [reply stock_picking_filter_lot/readme/USAGE.rst]: Ok that makes sense @lreficent.
Regarding the new features (I'd suggest a new dedicated PR):
* The available quantity to transfer could be useful
* That's an interesting feature too, I'd add a check (maybe only visible in debug mode) in the lots popup that inherits from the configuration in the picking type and allows to override that for the current transfer

Comment (SimoRubi) [reply stock_picking_filter_lot/readme/USAGE.rst:5]: Ok that makes sense @lreficent.
Regarding the new features (I'd suggest a new dedicated PR):
* The available quantity to transfer could be useful
* That's an interesting feature too, I'd add a check (maybe only visible in debug mode) in the lots popup that inherits from the configuration in the picking type and allows to override that for the current transfer

**Review by SimoRubi — APPROVED** [review]
Just take a look at the semicolon I pointed out in https://github.com/OCA/stock-logistics-workflow/pull/467/commits/c64e3e450de99899570f9beda3352dd4b7a11432#r230679182, obviously not blocking

File: stock_picking_filter_lot/readme/USAGE.rst (hunk 1 above)
Comment (LoisRForgeFlow) [reply stock_picking_filter_lot/readme/USAGE.rst]: Done!

File: stock_picking_filter_lot/readme/USAGE.rst (hunk 2 above)
Comment (LoisRForgeFlow) [reply stock_picking_filter_lot/readme/USAGE.rst:5]: Ok, let's go for a different PR. I'll merge this when green then.

**Pull request merged**